daemon/src/maker_inc_connections.rs

```diff
@@ -113,10 +129,9 @@ impl Actor {
             FramedRead::new(read, wire::EncryptedJsonCodec::new(transport_state.clone()));
 
         let this = ctx.address().expect("self to be alive");
-        let taker_msg_channel = self.taker_msg_channel.clone_channel();
         let read_fut = async move {
             while let Ok(Some(msg)) = read.try_next().await {
-                let res = taker_msg_channel.send(FromTaker { taker_id, msg }).await;
+                let res = this.send(FromTaker { taker_id, msg }).await;
```

Comment:
Not sure if worth this indirection? Could `match` on `msg` here as well.

Reply:
That was my original intention, but at the time of handling a new connection the `setup_maker` actor doesn't exist. I think we would need to pass in a `Arc<Mutex<AddressMap>>` to this future, which would be clunky anyway.

Reply:
Ah yes, good point.